src/internal.c

```diff
@@ -35828,7 +35831,8 @@ static int DoSessionTicket(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
 #endif
 
 #ifdef OPENSSL_EXTRA
-        ssl->clSuites = clSuites;
+        ssl->clSuites = clSuites; /* cppcheck-suppress autoVariables
```

Comment:
Can you please expand the comment to include the info about `setting of a pointer in a non-local struct to a local stack allocation.`?

Reply:
:heavy_check_mark: